app/Helpers/ContactHelper.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App\Helpers;
+
+use App\Models\User\User;
+use Illuminate\Database\Eloquent\Builder;
+
+class ContactHelper
+{
+    /**
+     * Get the name order that will be used when rendered the Add/Edit forms
+     * about contacts.
+     *
+     * @param User $user
+     * @param Builder $query
+     * @return Builder
+     */
+    public static function orderContactQueryByUserPreference(User $user, $query): Builder
```

Comment:
In fact if you want to use Builder here, it's better to use a [scope](https://laravel.com/docs/8.x/eloquent#local-scopes)
See https://github.com/monicahq/monica/blob/master/app/Traits/Searchable.php